app/components/settings/profile-page/anonymous-mode-section.hbs

```diff
@@ -9,7 +9,12 @@
   </:belowDescription>
   <:default>
     <div class="inline-flex items-center gap-3">
-      <Toggle @isOn={{@user.hasAnonymousModeEnabled}} @isDisabled={{this.toggleIsDisabled}} {{on "click" this.handleToggle}} />
+      <Toggle
+        @isOn={{@user.hasAnonymousModeEnabled}}
+        @isDisabled={{this.toggleIsDisabled}}
+        {{on "click" this.handleToggle}}
+        data-test-profile-anonymous-mode-toggle
```

Comment:
```suggestion
        data-test-anonymous-mode-toggle
```

(consistent with other such usages like `-is-correct-toggle`, `extra-invoice-details-requested-toggle`)